docs/deployment/getting-started-locally/getting-started-locally.md

```diff
@@ -72,19 +73,19 @@ This is similar to `make deploy-dev` but additionally configures containers in p
 
 Go debugging uses [Delve](https://github.com/go-delve/delve). Please see the [skaffold debugging documentation](https://skaffold.dev/docs/workflows/debug/) how to setup your IDE accordingly.
 
-!!! note
-    Resuming or stopping only a single goroutine (Go Issue [25578](https://github.com/golang/go/issues/25578), [31132](https://github.com/golang/go/issues/31132)) is currently not supported, so the action will cause all the goroutines to get activated or paused.
+> [!NOTE]
```

Comment:
I'm not sure if this works with MkDocs. From the documentation (https://squidfunk.github.io/mkdocs-material/reference/admonitions/) it seems like only the `!!! note` variety is supported. Could you please double-check this?

Reply:
This is what other gardener repositories are using for markdown syntax, see also the official documentation https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Yes, but the `etcd-druid` repository uses MkDocs to render the documentation at https://gardener.github.io/etcd-druid/. This is unsupported syntax in MkDocs.